File: identity-backend/src/main/java/io/github/ma1uta/identity/api/Status.java
package io.github.ma1uta.identity.api;

import io.github.ma1uta.matrix.EmptyResponse;
import io.github.ma1uta.matrix.identity.api.StatusApi;

/**
 * Implementation of the {@link StatusApi}.
 */
public class Status implements StatusApi {
    @Override
    public EmptyResponse v1Status() {
        return new EmptyResponse();
    }
}

File: identity-server/src/main/kotlin/io/github/ma1uta/identity/api/Status.kt
package io.github.ma1uta.identity.api

import io.github.ma1uta.matrix.EmptyResponse
import io.github.ma1uta.matrix.identity.api.StatusApi
import org.springframework.stereotype.Component

@Component
class Status : StatusApi {
    override fun v1Status() = EmptyResponse()
}
